grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -11,11 +11,11 @@ package org.pih.warehouse.reporting
 
 import grails.converters.JSON
 import grails.plugin.springcache.annotations.CacheFlush
-import grails.plugin.springcache.annotations.Cacheable
 import org.apache.commons.lang.StringEscapeUtils
 import org.pih.warehouse.core.Location
 import org.pih.warehouse.inventory.Transaction
 import org.pih.warehouse.report.ChecklistReportCommand
+import org.pih.warehouse.report.InventoryByLocationReportCommand
```

Comment:
This is fine, but you could have reused the InventoryReportCommand, just added a nullable List/Set of Locations. We should rename this to something like MultipleLocationInventoryReportCommand.

MultiLocationInventoryReportCommand? I don't know. I hate naming things. I just don't like "By" in class names.

Reply:
Besides my comments in the ticket, this looks good. I'll let Kelsey and Jesse review whether the report does what they expect. Using the inventory_snapshot table was probably a good choice from a performance standpoint, but we should be making requests dynamically through a REST API instead of doing this as a SSR'd report.
```
inventoryApi.fetchInventoryData(locations)
```
Anyway, we can refactor late once we have more discussions about the broader reporting framework UI.

Reply:
@jmiranda I'm not sure what do you mean, should I rename the InventoryByLocationReportCommand to MultiLocationInventoryReportCommand or reuse and rename the InventoryReportCommand?
If I reuse the InventoryReportCommand then should I add a new field for the result `Map<Location, Integer>`  or reuse one of the existing fields: quantityTransferredInByLocation or quantityTransferredOutByLocation?

Reply:
@pmuchowski sorry about being vague. yes, let's change the name to MultiLocationInventoryReportCommand. The rest of the suggestions were just brainstorming how I would like to think about these features going forward.

If we were going to reuse InventoryReportCommand, you could added a new association to that class. But your instinct not to re-use was correct as it would have probably caused confusion.